tests/mainnet/beacon_chain/historical_summaries_with_proof/deneb/historical_summaries_with_proof.yaml

```diff
@@ -0,0 +1,14 @@
+# Beacon state is sourced from consensus-spec-tests: https://github.com/ethereum/consensus-spec-tests/blob/7ffca8d9ec30d937bef52097be6715419e93a4ba/tests/mainnet/deneb/ssz_static/BeaconState/ssz_random/case_0/value.yaml
+content_key:
+  - 0x14ae7e346485874006
+content_value:
+  - 0x6a95a1a9ae7e346485874006ac0000006f4ea1cbdfedfaf60000000000000000000000000000000000000000000000008fa1cfc27730c1942e4919e2a8bd2668e5ba02ee7769599a931db20dfeb7120cdb56114e00fdd4c1f85c892bf35ac9a89289aaecb1ebd0a96cde606a748b5d719cc5feb29e02a89a7429376ff532621801239f408e7f7e63049a771d5b6473dc5a3afe60235feefc3e97234390b780e1996cbee7439111ca287fa001a9e1cb7537c8d6fb365cbcb3c68b958881578ef49bb8b51440c6b934e8e9dc8366c9f4136bba5413b4948f8015a1abe9e3df190a0d0ba524facfa9340f310a5af718d93311dd263279526c4f8575a52cc6eace34090dbd6d534ab44fad8351757dea160a993f8d25ef03a1096a86bcc10371a3f0b83d1f322532723202f021e1b95f2d5060a6e9eb9c8d35d5c9644fea283bfd553fddf6613069050a4c1ac38047df75aa3fa064c5de29b76edf1cde4fde3ac58d203d7ba87165cbb542fb7a7967565929
```

Comment:
I have issues with decoding this, I get invalid offset for the dynamic portion (the summaries list). It also seems that the epoch number of not correct? I assume the epoch provided lower is supposed to be the same as the one in the object?

Reply:
Actually, it looks correct. I didn't realize/forgot we also specified a fork digest in front for the `HistoricalSummariesWithProof`.

Reply:
Added the fork digest and I can decode it now. 

However, there is some ambiguity. Because the state was sourced from the link in consensus-spec-tests I had first created the ForkDigests with the `genesis_validators_root = 0x2170688a9e92595fb353c0a2ad6733431a8066c7ecb48ab3b2aaf9091a1722b1`. Decoding would fail with invalid fork however.

Seems that the ForkDigests used are the ones of mainnet. When I use those values, decoding works.

Verification of the proof works also, which is most important.